docs/acm/component/server-foundation/metrics.md

```diff
@@ -3,15 +3,15 @@
+| acm_managed_cluster_count  | - | Managed cluster count | None |  Not exposed, STABLE |
+| acm_managed_cluster_addon_status_condition | gauge | Managed cluster addon status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`addon_name`</td> <td>The name of the add-on.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The name of the managed cluster to which the add-on belongs.</td> </tr> <tr> <td>3</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster to which the add-on belongs.</td> </tr> <tr> <td>4</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>5</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions like `Available`, the status is set to unknown if the condition is missing. </td> </tr></table> | Not exposed |
+| acm_managed_cluster_status_condition | gauge | Managed cluster status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an {ocp-short} cluster, otherwise it is the cluster name.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster.</td> </tr> <tr> <td>3</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>4</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions like `ManagedClusterConditionAvailable`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
+| acm_manifestwork_status_condition | gauge | Manifestwork status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`manifestwork`</td> <td>The name of the manifestwork.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The name of the managed cluster to which the manifestwork belongs.</td> </tr> <tr> <td>3</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster to which the manifestwork belongs.</td> </tr> <tr> <td>4</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>5</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions like `Applied` and `Available`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
```

Comment:
Hard to pinpoint the exact spot in this long line but make sure to use "such as" instead of "like" when listing specific examples.

Reply:
Thanks. I picked those up in another PR.